simulation/README.md

```diff
@@ -32,6 +33,11 @@ simulation:
 
 `data_file`: This is the file where all recorded events are stored.
 
+`callers`: is a list of event sources for recording of events. When an item is changed, the change is done
+by someone, e.g. knx for changes from the bus. The canner name is identical with the plugin name that changes the item.

```

Comment:
I think the `caller` is not the plugin name. It is something the plugin developer decide. The KNX plugin is using `KNX`, but registering a plugin multiple times you are not able to ecide if it was from the first or second plugin instance.

Usually the plugin updates the item by just calling `item(value, caller, ...)`. The caller parameter can be anything. Just an idea: we could introduce a method in plugins superclass which updates an item and always uses the plugin instance name as caller value.